### PR TITLE
Fixing guard for getopts

### PR DESCRIPTION
**Pull request by elbing**
Signed-off-by: Álvaro Jurado <elbingmiss@gmail.com>

### CODE REVIEW

File: include/unistd.h [hunk 1]
```diff
@@ -170,12 +170,14 @@ extern int tcsetpgrp(int, pid_t);
...
 #endif
 
-/* berkeley specific functions */
-extern int	getopt(int, char* const*, const char*);
-extern int	opterr;
-extern int	optind;
-extern int	optopt;
-extern char	*optarg;
+/* 1003.2-1992 */
+#ifndef _GETOPT_DECLARED
```
Comment (gmacd) [inline include/unistd.h]: Maybe better to name it something like HARVEY_GETOPT_DECLARED?

Comment (elbing) [reply include/unistd.h]: POSIX compliance, that's for -D_GETOPT_DECLARED definition. Many configure scripts will check that.

**Comment by elbing** [comment]
I had the local branch in this machine without env script. Rebased.

**Comment by elbing** [comment]
Nope! musl getopt.h hasn't the guard. This another problem in gcc/libiberty.

**Pull request closed without merging**